.vscode/settings.json

```diff
@@ -53,5 +53,24 @@
         "editor.defaultFormatter": "esbenp.prettier-vscode",
         "editor.wordWrap": "on",
         "editor.quickSuggestions": true
-    }
+    },
+    "i18n-ally.localesPaths": [
+        "packages/dashboard/src/locales",
```

Comment:
Please move this to another PR and integrate this with `npx gulp new-pkg` script.